pac4j-kerberos/src/test/java/org/pac4j/kerberos/client/direct/KerberosClientKerbyTest.java

```diff
@@ -0,0 +1,97 @@
+package org.pac4j.kerberos.client.direct;
+
+import org.apache.kerby.kerberos.kdc.impl.NettyKdcServerImpl;
+import org.apache.kerby.kerberos.kerb.KrbException;
+import org.apache.kerby.kerberos.kerb.server.SimpleKdcServer;
+import org.junit.Before;
+import org.junit.Test;
+import org.pac4j.core.context.HttpConstants;
+import org.pac4j.core.context.MockWebContext;
+import org.pac4j.core.util.TestsConstants;
+import org.pac4j.kerberos.credentials.KerberosCredentials;
+import org.pac4j.kerberos.credentials.authenticator.KerberosAuthenticator;
+import org.pac4j.kerberos.credentials.authenticator.SunJaasKerberosTicketValidator;
+import org.pac4j.kerberos.profile.KerberosProfile;
+import org.springframework.core.io.FileSystemResource;
+
+import java.io.File;
+import java.io.IOException;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+/**
+ * This is a test with real kerberos ticket validation:
+ * - spins up a mini/lightweight Kerberos server (using Apache Kerby)
+ * - generates the keytabs for service and client
+ * - generates a real service ticket, and passes it to mock context
+ * - checks that the ticket is correctly validated, and yields a correct client ID
+ *
+ * @author Vidmantas Zemleris, at Kensu.io
+ */
+public class KerberosClientKerbyTest implements TestsConstants {
```

Comment:
it's somewhere between integration and unit test.
likely I should make it run together with the integration tests...